modules/apps/object/object-service/src/main/java/com/liferay/object/internal/system/OrganizationSystemObjectDefinitionManager.java

```diff
@@ -0,0 +1,193 @@
+import com.liferay.petra.sql.dsl.Column;
+import com.liferay.petra.sql.dsl.Table;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.BaseModel;
+import com.liferay.portal.kernel.model.OrganizationTable;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.service.OrganizationLocalService;
+import com.liferay.portal.kernel.util.GetterUtil;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Mateus Santana
+ */
+@Component(enabled = false, service = SystemObjectDefinitionManager.class)
+public class OrganizationSystemObjectDefinitionManager
+	extends BaseSystemObjectDefinitionManager {
+
+	@Override
+	public long addBaseModel(User user, Map<String, Object> values)
+		throws Exception {
+
+		OrganizationResource organizationResource = _buildOrganizationResource(
+			user);
+
+		Organization organization = organizationResource.postOrganization(
+			_toOrganization(values));
+
+		setExtendedProperties(
+			Organization.class.getName(), organization, user, values);
+
+		return GetterUtil.getLong(organization.getId());
+	}
+
+	@Override
+	public BaseModel<?> deleteBaseModel(BaseModel<?> baseModel)
+		throws PortalException {
+
+		return _organizationLocalService.deleteOrganization(
+			(com.liferay.portal.kernel.model.Organization)baseModel);
+	}
+
+	@Override
+	public BaseModel<?> fetchBaseModelByExternalReferenceCode(
+		String externalReferenceCode, long companyId) {
+
+		return _organizationLocalService.
+			fetchOrganizationByExternalReferenceCode(
+				externalReferenceCode, companyId);
+	}
+
+	@Override
+	public BaseModel<?> getBaseModelByExternalReferenceCode(
+			String externalReferenceCode, long companyId)
+		throws PortalException {
+
+		return _organizationLocalService.getOrganizationByExternalReferenceCode(
+			externalReferenceCode, companyId);
+	}
+
+	@Override
+	public String getExternalReferenceCode(long primaryKey)
+		throws PortalException {
+
+		com.liferay.portal.kernel.model.Organization organization =
+			_organizationLocalService.getOrganization(primaryKey);
+
+		return organization.getExternalReferenceCode();
+	}
+
+	@Override
+	public JaxRsApplicationDescriptor getJaxRsApplicationDescriptor() {
+		return new JaxRsApplicationDescriptor(
+			"Liferay.Headless.Admin.User", "headless-admin-user",
+			"organizations", "v1.0");
+	}
+
+	@Override
+	public Map<Locale, String> getLabelMap() {
+		return createLabelMap("organization");
+	}
+
+	@Override
+	public Class<?> getModelClass() {
+		return com.liferay.portal.kernel.model.Organization.class;
+	}
+
+	@Override
+	public List<ObjectField> getObjectFields() {
+		return Arrays.asList(
+			createObjectField(
+				"Text", "String", "comments", "comments", false, true),
```

Comment:
@mateussgn We have a mispattern here. No yours, but from portal.

In the DB, we have comments. In the headless API, we have comment. 
So please, follow the UserSystemObjectDefinitionManager implementation for this mispattern.

Also, implement the getVariables method to convert this mismatch.

Reply:
Sure!
Thanks for the tip!